### PR TITLE
Document testing

### PR DESCRIPTION
**Pull request by ubergesundheit**
This PR adds information about how this chart is being tested.

### CODE REVIEW

File: .github/pull_request_template.md
Comment (JosephSalisbury) [inline .github/pull_request_template.md:16]: Can we change to something like, required on chart changes etc. i.e - when would they be required?

File: README.md
Comment (JosephSalisbury) [inline README.md:169]:
```suggestion
Testing includes creation of`Deployment`, `Service`, and `Ingress` resources to check if reconciliation works as intended.
```

**Review by webwurst — APPROVED** [review]
Sounds good. I'll try the steps and add details if useful.

File: .github/pull_request_template.md
Comment (ubergesundheit) [reply .github/pull_request_template.md:16]: yes

File: .ats/main.yaml
Comment (webwurst) [inline .ats/main.yaml:14]: > iption
? ;D

**Comment by ubergesundheit** [comment]
Ok waiting on input from @webwurst before merge

**Review by webwurst — APPROVED** [review]
okidoki

**Comment by ubergesundheit** [comment]
I assume you did not found anything, otherwise just open a separate PR :)

**Pull request merged**
